ga4gh/datamodel/datasets.py

```diff
@@ -201,6 +246,7 @@ class FileSystemDataset(AbstractDataset):
     """
     variantsDirName = "variants"
     readsDirName = "reads"
+    featuresDirName = "sequenceAnnotations"
```

Comment:
Should we call the directory ``features`` for consistency? This structure isn't really user-facing, so it might be good to be consistent about our naming.